src/Illuminate/Auth/Passwords/PasswordBrokerManager.php

```diff
@@ -82,10 +82,8 @@ protected function resolve($name)
      */
     protected function createTokenRepository(array $config)
     {
-        $key = $this->app['config']['app.key'];
-
-        if (Str::startsWith($key, 'base64:')) {
-            $key = base64_decode(substr($key, 7));
```

Comment:
Look at here, we tested for `string starts with 'base64:' and then take the string after the first 7 characters`, the char count of base64: will never change...

Reply:
Reverted this